Libraries/Components/View/ViewPropTypes.js

```diff
@@ -452,6 +452,12 @@ export type ViewProps = $ReadOnly<{|
    */
   accessibilityHint?: ?Stringish,
 
+  /**
+   * Alias for accessibilityLabel  https://reactnative.dev/docs/view#accessibilitylabel
+   * https://github.com/facebook/react-native/issues/34424
+   */
+  'aria-label'?: ?Stringish,
+
```

Comment:
Type is Stringish here but string in the other components. Should we be using Stringish everywhere?

Reply:
Actually, I am giving types similar to `accessibilityLabel`.